src/idl_gen_general.cpp

```diff
@@ -1065,11 +1065,9 @@ void GenStruct(StructDef &struct_def, std::string *code_ptr) {
       }
     }
   // generate object accessors if is nested_flatbuffer
+  if (field.nested_flatbuffer != nullptr) {
   auto nested = field.attributes.Lookup("nested_flatbuffer");
```

Comment:
Like I said, this code looks incorrect. Why is it still looking up the attribute? Why is it getting the type from `nested->type.struct_def` instead of `field.nested_flatbuffer`? Are you running `generate_code.sh`, because I think `nested->type.struct_def` would be null?